Fix: Mage Talent Required SilverTongue2

suspi opened the pull request:
In the current build, Mage required Silver Tongue 2 instead of Silver Tongue 1 that lead up to it.

BlueWinds commented on src/content/intro/firstStorm.coffee:
```diff
@@ -218,7 +218,7 @@ Job.IntroSickNat::next = Page.IntroSickNat = class IntroSickNat extends Page
     <text continue><p><q>Magical drain. She must be quite powerful, to bring your ship through a storm like that.</q></p></text>
   </page>
   <page>
-    <text continue-inline> Seeing the confusion reflected in James' face, he finally deigned to explain. <q>Your captain here is a sorceress. And as I said, quite a powerful one at that. I wouldn't dare touch storm-energy. It's amazing she survived, but she'll make a full recovery, given enough time.</q></text>
```
This is actually fine as is. *James'* is the usually-preferred way to make a word ending in "s" possessive. http://www.quickanddirtytips.com/education/grammar/apostrophe-catastrophe-part-two?page=all

BlueWinds commented on src/content/intro/firstStorm.coffee:
```diff
@@ -241,7 +241,9 @@ Page.IntroSickNat::next = Page.IntroSickNat2 = class IntroSickNat2 extends Page
   <page>
-    <text continue-inline> Magic didn't manifest until puberty. They'd have been best friends for a decade when she learned what she was.</text>
+#	I changed "didn't" to "doesn't" for the reason that "didn't" implies that James was talking about Natalie's magic (i.e. "Her magic didn't manifest until puberty") while "doesn't" more implies that James was talking about how magic works in general (i.e. "This does that" over "This did that". The first one makes more sense).
+    <text continue-inline> Magic doesn't manifest until puberty. They'd have been best friends for a decade when she learned what she was.</text>
```
I'm going to cheat my way out of the ambiguity by changing it a little more - "Magic never manifested until puberty."

You're perfectly correct about "they'd have been" though. It's how I'd say it out loud, but it looks weird written.

BlueWinds commented on src/content/people/Kat/_preRoute.coffee:
```diff
@@ -170,7 +170,7 @@ Page.MeetKat3.next['Stolen purse?'] = Page.MeetKatMoney = class MeetKatMoney ext
     #{@Guard.image 'normal', 'far-right'}
     #{@Kat.image 'sad', 'right'}
-    <text continue><p>#{q}No,</q> Kat, finally spoke, tugging at her blouse. It looked both brand new and none too cheap. One of the shoulders was torn where Natalie had grabbed her. She stared at the rip a moment then started to cry silently. She hicced as the guard cuffed her ear.</p></text>
+    <text continue><p>#{q}No,</q> Kat, finally spoke, tugging at her blouse. It looked both brand new and none too cheap. One of the shoulders was torn where Natalie had grabbed her. She stared at the rip a moment then started to cry silently. She hiccuped as the guard cuffed her ear.</p></text>
```
It's not a dictionary word, but I like "hicced" better here and am going to leave it. Call it authorial stubbornness if you like. \*^^\*

BlueWinds commented:
I'm not great with git, so I just made all these edits in a separate commit (8064b09). If there's a better way to edit a PR before merging, I'm all ears. ^^;;

suspi committed:
Fix: Mage Talent Required SilverTongue2

In the current build, Mage required Silver Tongue 2 instead of Silver Tongue 1 that lead up to it.

BlueWinds commented:
Hah, whoops. Guess I've never tested far enough into the talent trees to notice. Thanks. :)
